### PR TITLE
New What's New for 0.8.2.

### PR DESCRIPTION
**Pull request by Racel**
### Purpose

Updated What's New Pages for 0.8.2. Please take a look and let me know if any changes need to be made.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR

### Reviewers
@ikeough 
@Benglin 
@RodRecker 
@riteshchandawar 

### FYIs
@jnealb

### CODE REVIEW

File: extern/gallery/en-US/GalleryContents.xml [hunk 1]
```diff
@@ -2,29 +2,34 @@
...
+		<Header>Save Design Options with Presets</Header>
+		<Body>Presets allow you to keep track of different states of your graph by capturing the values of its input nodes.</Body>
+		<ImagePath>Presets.png</ImagePath>
 	</GalleryContent>
 	<GalleryContent>
-		<Header>Crash Recovery</Header>
-		<Body>All your working files are now safe even when Dynamo crashes. Look out for recovered files in the start page when you restart Dynamo after a crash.</Body>
-		<ImagePath>crash_recovery.png</ImagePath>
+		<Header>Convert Nodes to Code</Header>
+		<Body>Select groups of nodes and automatically generate the corresponding Design Script code.</Body>
```
Comment (lukechurch) [inline extern/gallery/en-US/GalleryContents.xml]: DesignScript is one word :)

**Comment by RodRecker** [comment]
@Racel Can you add a note about command line usage? Maybe add it on the last page.

**Comment by holyjewsus** [comment]
@RodRecker @Racel the documentation is here on the wiki
https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Command-Line-Interface

**Comment by Racel** [comment]
@RodRecker -Added a line about it on the last page.

**Comment by RodRecker** [comment]
@Racel looks good - thank you!

**Comment by Racel** [comment]
@RodRecker - Made a minor change after some discussion with @holyjewsus. Let me know if I can go ahead and merge with master.

**Comment by Benglin** [comment]
Thanks for the work, @Racel, it looks good to me. When it comes to testing, I'd be interested to see how the new installer remove older gallery files (e.g. `%ProgramData%\Dynamo\0.8\gallery\Data\multiple_previews.png`). I'm sure @jnealb or @riteshchandawar can see if that works.

**Comment by Benglin** [comment]
On a second thought, these images contain various text contents in them, which means they will look odd on other languages. there are two ways around this:

1. Move the images into `extern/gallery/en-US` folder, and they will be localized by WWL team.
2. Update images to contain lesser (and language agnostic contents) or no text,

**Comment by riteshchandawar** [comment]
@Racel Publish Local is missing for what's new section.
image


In-Canvas search was delivered in 0.8.1 as well. We need to showcase now In-Canvas Search have capabilities of Tooltip and keyboard navigation keys.

Thanks,
Ritesh

**Comment by Racel** [comment]
@Benglin @riteshchandawar:
 - I moved the images to extern/gallery/en-US for localization. FYI - previous What's New slides contain English text also, though. I used current slides as a template.
 - Added some text to the 4th slide indicating improved keyboard navigation for search
 - Added image/text related to Publish Local

Please let me know if it is ok now.

**Comment by RodRecker** [comment]
@monikaprabhu Can you check that the localization team can handle these images?

File: extern/gallery/en-US/GalleryContents.xml [hunk 2]
```diff
@@ -2,29 +2,34 @@
...
+		<Header>Improved In-Canvas Search and Tooltips</Header>
+		<Body>Search algorithms and keyboard navigation have been improved so that you can quickly search for nodes without using the Library.&#10;Right-click (or Shift + Double-click) over the canvas to bring up search. Hovering over search results reveals helpful tooltips.</Body>
+		<ImagePath>In-CanvasSearch.png</ImagePath>
 	</GalleryContent>
 	<GalleryContent>
-		<Header>Draggable Library Items</Header>
-		<Body>You can now drag items from the library. It helps you place nodes precisely on the first attempt.</Body>
-		<ImagePath>library_draggable.png</ImagePath>
+		<Header>Color on Surfaces</Header>
+		<Body>Smooth color shading can now be applied to surfaces with the Display.BySurfaceColors node.</Body>
```
Comment (ikeough) [inline extern/gallery/en-US/GalleryContents.xml]: We should not use the term "Smooth color shading". In 3D, smooth shading is shading that incorporates the surface normal into its coloration calculations. This is in contrast to flat shading which only considers the triangle normal.

Suggest:
"Square arrays of color can now be applied..."

**Comment by Racel** [comment]
Thanks, @ikeough. I updated the text and added another sentence about applying images. Please take a look.

**Comment by Racel** [comment]
@Benglin - can you tell me how to change '0.8' at the top of the what's new template to '0.8.2'?

**Comment by Benglin** [comment]
@Racel, I figured it takes less time for me to code/test it than to put a description here making sure things are covered. You can cherry-pick [this commit from my repository](https://github.com/DynamoDS/Dynamo/commit/009464db12f1f175602263bec9a7ba786ac203d5) instead :)

**Comment by Racel** [comment]
@Benglin - 
Thanks for the commit. We will cherry-pick that.

@ikeough @RodRecker @Benglin @riteshchandawar - Updated the images based off of @Benglin's suggestions. The text in the images has been reduced for better localization practices. 

@ikeough - got rid of the extra sentence in the 'Color on Surfaces' slide.

Let me know if this is ready for cherry-picking. We will need to grab this and @Benglin's commit above.

**Comment by ikeough** [comment]
LGTM. Thanks @Racel!

**Comment by Racel** [comment]
@Benglin - added your commit to this PR. Did not want to have to cherry pick them separately.

**Comment by riteshchandawar** [comment]
@lukechurch @Racel We need to cherry pick this to RC 082 brach as well.

- Ritesh

**Comment by Racel** [comment]
@riteshchandawar - we cherry picked it for the RC 082 yesterday. The commit on RC 082 includes ben's commit.

**Comment by riteshchandawar** [comment]
Thanks @Racel

**Comment by Benglin** [comment]
I like your new images very much, @Racel, awesome stuff! :+1:

**Pull request merged**